docs/raindex/example-strats/2-recharging-grid-strategy.md

```diff
@@ -0,0 +1,161 @@
+
+For example, to recharge tranches once per day:
+
+```
+seconds in a day = 60 * 60 * 24 = 86400
+recharge rate = 1e18 / 86400 = 11574e9
+```
+
+What does this mean in practice? Consider the example above, where we have initialised our strategy in tranche 6, which means we have tokens available for $1.60. The market starts at $1.57, and for the first 24 hours it hovers at that value, edging up to just under $1.60.
+
+Tranches always recharge **in order** back to the first tranche, so immediately tranche 5 starts to recharge (ignore the delay for now, explained below).
+
+This means that over the course of the day, as that tranche recharges, there will be arbitrage opportunites and we'll see a full tranche of tokens clear at $1.50.
+
+### Recharge delay
+After a tranche is fully cleared, there is a configurable delay before the tranche starts recharging.
+
+This pause enables the strategy to offer the next tranche at a different price, adapting to market conditions. Tokens from two tranches cannot clear in a single trade, so at the tranche boundary we need to provide some time to allow the market to move into the next tranche, before recharging the previous one.
+
+The default delay is set to 5 minutes (300 seconds), which generally provides a good balance between allowing a reaction to the market, and maintaining fluid trading.
+
+### Shy liquidity
+When a tranche is fully cleared, the strategy may introduce the next tranche partially, a concept known as "shy liquidity." Here’s how it works:
+
+- Partial Activation: Instead of immediately offering the full amount of the next tranche, a fraction is made available. For example, if tranches are set to be 90% shy (i.e., 9e17), clearing a tranche completely will start the next tranche at 10% of its maximum size. This gradual introduction helps manage liquidity more efficiently.
+- Capital Efficiency: By only partially activating the next tranche, the strategy reduces the capital requirements needed to adjust prices as the market moves.
+
+Shyness Setting: The shyness parameter can be adjusted:
+- Non-zero Shyness: If set to a value less than 1e18, it ensures that only a portion of the tranche is initially available. For example, with 90% shyness, only 10% of the next tranche is activated upon clearing the previous one.
+- Zero Shyness: If set to 0, each tranche will be fully available as soon as it is entered.
+
+It’s important to note that shyness must be less than 1e18 to avoid skipping tranches entirely.
+
+### Minimum trade size
+Because clearing a tranche causes a delay, somebody could grief the strategy by clearing the smallest possible trade every time the current tranche started to recharge. To mitigate this, we can set a minimum trade size.
+
+## Parameters
+
+At the time of writing, specific variables that can be configured in [tranche-space.rain](https://github.com/rainlanguage/rain.dex.pubstrats/blob/main/src/tranche-space.rain) are below. 
```

Comment:
do we need to keep these up to date as we update pubstrats? there's no process for that atm

Reply:
https://github.com/rainlanguage/rain.dex.pubstrats/issues/50